lib/util/crypto.js

```diff
@@ -43,6 +43,8 @@ const generateToken = (bytes = 48) => randomBytes(bytes).toString('base64')
   .replace(/\//g, '!')
   .replace(/\+/g, '$');
 
+const isValidStandardLengthToken = (token) => /^[A-Za-z0-9!$]{64}$/.test(token);
```

Comment:
What do you think about a name that's a little shorter? How about `isValidToken` or `matchesTokenFormat`?

Reply:
I think `isValidToken()` would be fine, except that this is a valid token:

```js
require('./lib/util/crypto').generateToken(0)
''
```

In the current codebase, there are zero uses of `generateToken()` with an argument, so to allow `isValidToken` as the name for this function, I think step 1 would be to remove the `bytes` argument from `generateToken()`.  Would that be an acceptable change?

Reply:
PR to remove `bytes` arg at git push --set-upstream origin generate-token-no-args

Reply:
Just approved #1264!